docs/admin/cluster-management.md

```diff
@@ -14,47 +14,141 @@ certainly want the docs that go with that version.</h1>
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Cluster Management
 
-This doc is in progress.
+## Creating and configuring a Cluster
+
+Assuming that you have machines for your clusters ready, you need to install Kubernetes on them. For specific instructions you can consult one of existing [Getting Started guides](../../docs/getting-started-guides/README.md) for various environments:
+* GKE [GKE page](https://cloud.google.com/container-engine),
+* Vagrant [Vagrant setup](../../docs/getting-started-guides/vagrant.md),
+* GCE [GCE setup](../../docs/getting-started-guides/gce.md),
+* CoreOS on Azure [CoreOS on Azure setup](../../docs/getting-started-guides/coreos/azure/README.md),
+* CoreOS on AWS [CoreOS on AWS setup](../../docs/getting-started-guides/coreos.md),
+* CoreOS on VMWare [CoreOS on VMWare setup](../../docs/getting-started-guides/coreos.md).
+
+For more comprehensive and detailed description of cluster setup and configuration process see [From Scratch](../../docs/getting-started-guides/scratch.md) guide.
+
+## Tools for interacting with Kubernetes
```

Comment:
I think you should remove this section. The first paragraph is not needed because we describe kuebctl in the user guide, and the getting started guides explain when/how to use the relevant commands from cluster/. The stuff in the second and third paragraph is good but should be merged into cluster/README.md, not put in this doc.